### ISSUE TITLE
Parity browser UI is deprecated.

### ISSUE THREAD

**Issue by stone212** [paritytech/parity#8969]
_Before filing a new issue, please **provide the following information**._

> I'm running:
>
> - **Which Parity version?**: 1.10
> - **Which operating system?**: Debian
> - **How installed?**: via binaries
> - **Are you fully synchronized?**: yes
> - **Which network are you connected to?**: ethereum
> - **Did you try to restart the node?**: yes

_Your issue description goes here below. Try to include **actual** vs. **expected behavior** and **steps to reproduce** the issue._

After upgrading to 1.10 stable, when I go to my Parity UI address at 127.0.0.1:8180 I get the error:

Parity browser UI is deprecated.

Obviously the UI has been made a separate software.  And I have installed that software on the node.

Going to the Wiki and searching for "Parity UI" does not tell me how to start and use the new software.  Please direct me to new pages to explain how to use the Parity UI.  I am sorry if this has already been discussed.

**Comment by 5chdn** [comment]
Hi Stone,

this https://wiki.parity.io/Parity-Wallet

**Comment by stone212** [comment]
@5chdn  Thank you for sending that link.  It is strange that that doesn't come up in search (https://wiki.parity.io/index.html?q=Parity+UI)

But it doesn't have the answer I need.  If you interpret my question as "How do I start the Parity UI" and that link says "launch Parity UI" with no instructions or link then I don't have an answer.  

After some trial and error I find that there is a command called "parity-ui" so I tried that.

Then I manually installed libasound2 because the .deb didn't do that automatically for some reason (I'll open an Issue for that) and then I get this error:

`(parity-ui:1660): Gtk-WARNING **: cannot open display:`

I will open that Issue also.
